.github/workflows/deploy_preview.yml

```
@@ -0,0 +1,33 @@
+name: Deploy Preview to Netlify
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy-preview:
+    name: Build and Deploy preview on Netlify
+    runs-on: ubuntu-latest
```

could we move the `timeout-minutes: 5` here, it's more visible here than at the end of the file

```suggestion
    runs-on: ubuntu-latest
    timeout-minutes: 5
```

---

@tyrro it's the timeout for the deployment step, not the whole workflow. So I think it's better keep it there 🙏